### ISSUE TITLE
Error in loading item ytsearch:

### ISSUE THREAD

**Issue by Mrtol** [sedmelluq/lavaplayer#517]
```
21:00:24.145 [lava-daemon-pool-info-loader-2-thread-1] ERROR com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager - Error in loading item ytsearch: sis
com.sedmelluq.discord.lavaplayer.tools.FriendlyException: Something went wrong when looking up the track
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:43)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.dispatchItemLoadFailure(DefaultAudioPlayerManager.java:237)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:226)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.executeQueue(OrderedExecutor.java:98)
	at com.sedmelluq.discord.lavaplayer.tools.OrderedExecutor$ChannelRunnable.run(OrderedExecutor.java:87)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.lang.RuntimeException: java.lang.NullPointerException: Cannot invoke "String.split(String)" because "durationText" is null
	at com.sedmelluq.discord.lavaplayer.tools.ExceptionTools.wrapUnfriendlyExceptions(ExceptionTools.java:57)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.loadSearchResult(YoutubeSearchProvider.java:74)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.search(YoutubeAudioSourceManager.java:254)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager$LoadingRoutes.search(YoutubeAudioSourceManager.java:220)
	at com.sedmelluq.discord.lavaplayer.source.youtube.DefaultYoutubeLinkRouter.route(DefaultYoutubeLinkRouter.java:34)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItemOnce(YoutubeAudioSourceManager.java:188)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.loadItem(YoutubeAudioSourceManager.java:127)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItemOnce(DefaultAudioPlayerManager.java:437)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.checkSourcesForItem(DefaultAudioPlayerManager.java:419)
	at com.sedmelluq.discord.lavaplayer.player.DefaultAudioPlayerManager.lambda$createItemLoader$0(DefaultAudioPlayerManager.java:218)
	... 6 common frames omitted
Caused by: java.lang.NullPointerException: Cannot invoke "String.split(String)" because "durationText" is null
	at com.sedmelluq.discord.lavaplayer.tools.DataFormatTools.durationTextToMillis(DataFormatTools.java:91)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.extractPolymerData(YoutubeSearchProvider.java:161)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.lambda$polymerExtractTracks$0(YoutubeSearchProvider.java:149)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.polymerExtractTracks(YoutubeSearchProvider.java:148)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.extractSearchResults(YoutubeSearchProvider.java:94)
	at com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider.loadSearchResult(YoutubeSearchProvider.java:71)
	... 14 common frames omitted
```

**Comment by MCOfficer** [comment]
please provide more information, such as LP version, youtube link etc.

**Comment by Mrtol** [comment]
> lütfen LP sürümü, youtube bağlantısı vb. gibi daha fazla bilgi sağlayın.

![image](https://user-images.githubusercontent.com/41306224/92389500-e1a31e80-f121-11ea-9214-10567e6a3085.png)

No matter what the music is, the same music sometimes works, sometimes it fails

**Comment by Mrtol** [comment]
![image](https://user-images.githubusercontent.com/41306224/92389886-93424f80-f122-11ea-879e-94748b2c8be7.png)

instead

 ```
public static long durationTextToMillis(String durationText) {

        int length = 0;

        if (durationText == null) {
            
            return length;
        }

        for (String part : durationText.split("[:.]")) {
            length = length * 60 + Integer.valueOf(part);
        }

        return length * 1000L;
    }
```

**Comment by HappyAreaBean** [comment]
I am having this issue too.

**Comment by caneleex** [comment]
you have to use commit [bec3995](https://github.com/sedmelluq/lavaplayer/commit/bec39953a037b318663fad76873fbab9ce13c033) as the lavaplayer version as sed hasn't released a new release which fixes this. you have to add jitpack.io as your maven repo url to be able to use the commit.

gradle example:
```gradle
repositories {
    jcenter() // repo for other dependencies
    maven { url 'https://jitpack.io' }
}

dependencies {
   implementation group: 'com.sedmelluq', name: 'lavaplayer', version: 'bec3995'
}
```

**Comment by Mrtol** [comment]
> you have to use commit [bec3995](https://github.com/sedmelluq/lavaplayer/commit/bec39953a037b318663fad76873fbab9ce13c033) as the lavaplayer version as sed hasn't released a new release which fixes this (#495). you have to add jitpack.io as your maven repo to be able to use the commit (or respectively any commit) as the version.
> 
> gradle example:
> 
> ```groovy-gradle
> repositories {
>     jcenter() // repo for other dependencies
>     maven { url 'https://jitpack.io' } // required to be able to use commits as dependencies' version
> }
> 
> dependencies {
>    implementation group: 'com.sedmelluq', name: 'lavaplayer', version: 'bec3995'
> }
> ```
![image](https://user-images.githubusercontent.com/41306224/93468650-f9378f80-f8f7-11ea-9fab-15ebfbebc487.png)

![image](https://user-images.githubusercontent.com/41306224/93468677-02c0f780-f8f8-11ea-85b8-c43b79d88696.png)

it doesn't work in maven

**Comment by caneleex** [comment]
> it doesn't work in maven
i forgot you have to use `com.github.sedmelluq` as the group (in gradle) or group id (in maven)
